driver-core/src/test/java/com/datastax/driver/core/CCMBridge.java

```diff
@@ -1047,13 +1047,15 @@ public CCMBridge build() {
         cassandraConfiguration.remove("rpc_port");
         cassandraConfiguration.remove("thrift_prepared_statements_cache_size_mb");
       }
-      if (isMaterializedViewsDisabledByDefault(cassandraVersion)) {
-        // enable materialized views
-        cassandraConfiguration.put("enable_materialized_views", true);
-      }
-      if (isSasiConfigEnablementRequired(cassandraVersion)) {
-        // enable SASI indexing in config (disabled by default in C* 4.0)
-        cassandraConfiguration.put("enable_sasi_indexes", true);
+      if (!dse) {
```

Comment:
Perhaps the DSE check should be folded into isMaterializedViewsDisabledByDefault() and isSasiConfigEnablementRequired() instead?  Seems like that would make those two methods more accurate in general.

Reply:
That was actually my first version of this snippet :-) But the implementations felt clumsy to me as both would start with a `if (dse) return false;` statement that seemed disconnected from the rest. Anyways, it doesn't matter much, there is no version of DSE as of today that support these config options.

Reply:
It's the fact that no DSE version uses these configs that formed the foundation of my concern... that's why you want those methods to be as accurate as possible.  But we're talking about private methods in CCMBridge so it's hard to get too concerned... this also falls squarely under the category of "nit".